simulator.code-workspace

```diff
@@ -64,9 +68,9 @@
 				"name": "Debug LVGL demo with gdb",
 				"type": "cppdbg",
 				"request": "launch",
-			        "program": "${workspaceFolder}/build/bin/demo",
+			        "program": "bin/main",
```

Comment:
I have to change it as below to let it works for me:
"program": "${workspaceFolder}/bin/main",
"MIMode": "gdb",

Also need to add debug mode in the CmakeLists.txt:
set(CMAKE_BUILD_TYPE Debug)

Reply:
you are right, it works perfect with "cmake::Select Variant"!